boards/arm/nrf52840_papyr/doc/nrf52840_papyr.rst

```diff
@@ -0,0 +1,193 @@
+* RADIO (Bluetooth Low Energy and 802.15.4)
+* :abbr:`RTC (nRF RTC System Clock)`
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UART (Universal asynchronous receiver-transmitter)`
+* :abbr:`USB (Universal Serial Bus)`
+* :abbr:`WDT (Watchdog Timer)`
+* COUNTER
+
+.. figure:: img/nrf52840_papyr.jpg
+     :width: 442px
+     :align: center
+     :alt: Electronut Labs Papyr
+
+     Electronut Labs Papyr (Credit: Electronut Labs)
+
+More information about the board is available at https://docs.electronut.in/papyr/.
+
+Hardware
+********
+
+Papyr has two external oscillators. The frequency of
+the slow clock is 32.768 kHz. The frequency of the main clock
+is 32 MHz.
+
+Supported Features
+==================
+
+The nrf52840_papyr board configuration supports the following
+hardware features currently:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| ADC       | on-chip    | adc                  |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth,           |
+|           |            | ieee802154           |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| USB       | on-chip    | usb                  |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED1 (green) = P0.13
+* LED2 (blue)  = P0.15
+* LED3 (red)   = P0.14
+
+Push buttons
+------------
+
+* Reset = SW0 = P0.18 (can be used as GPIO also)
+
+UART
+----
+
+* TX = P0.8
+* RX = P0.7
+
+I2C
+---
+
+I2C pins connected to onboard sensors:
+
+* SDA = P0.5
+* SCL = P0.6
+
+SPI
+---
+
+The e-paper display is connected to the chip via SPI on the following pins (SPI_0):
+
+* SCK  = P0.31
+* MOSI = P0.29
+* MISO = P0.33 (not used by the display)
```

Comment:
P0 has 32 GPIOs (P0.0-P0.31). This should be probably P1.1 (pin 33 in absolute numeration).